DisplaySettings/DisplaySettings.cpp

```diff
@@ -408,8 +414,11 @@ namespace WPEFramework {
 
         void DisplaySettings::Deinitialize(PluginHost::IShell* /* service */)
         {
+	   LOGERR("Enetering DisplaySettings::Deinitialize");
```

Comment:
Please, consider changing this to LOGINFO().

Reply:
it is backport, this should not be changed